content/articles/2019-03-tibble-2-1-1.Rmarkdown

```diff
@@ -0,0 +1,87 @@
+---
+title: tibble 2.1.1
+slug: tibble-2.1.1
+description: >
+    Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! This article describes and motivates the latest minor release.
```

Comment:
I think it's better to list the main changes here, rather than recapitulating what the package does.